src/fuzzers/FuzzBuffer.cc

```diff
@@ -2,6 +2,10 @@
 #define _GNU_SOURCE
 #endif
 
+#ifdef _MSC_VER
+#include <mem.h>
+#endif
```

Comment:
This block should go down underneath the #include below it, per the style guide.

Reply:
done